packages/forms-web-app/src/controllers/your-planning-appeal.js

```diff
@@ -0,0 +1,21 @@
+const logger = require('../lib/logger');
+const { VIEW } = require('../lib/views');
+const appealModel = require('../models/appealModel');
+
+exports.getYourPlanningAppeal = async (req, res, next) => {
+  const { appealId } = req.params;
+  let appeal;
+
+  try {
+    appeal = await appealModel.getAppeal(appealId);
```

Comment:
There is existing code to do this via middleware. 

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/middleware/fetch-appeal-by-url-param.js

An example of how it's used:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/routes/appellant-submission/submission-information.js#L11

If you prefer your approach, please can you refactor the above & any other usages to use your code, so we keep one standard?

Reply:
Cheers Chris.. .I saw this but the functioanlity/behaviour wanst quite what was needed I think. Will have a chat and go from there.